package-lock.json

```diff
@@ -8437,11 +9060,96 @@
             "integrity": "sha512-fpWrvyVHEKyeEvbKZTVOeZF3VSKKWtJxFIxX/jaVPf+cLbGUSitjb49pHLqPV2BUNNZ0LcoeEGfE/YCpyDYHIw==",
             "dev": true
         },
+        "node_modules/babel-jest": {
+            "version": "29.7.0",
+            "resolved": "https://registry.npmjs.org/babel-jest/-/babel-jest-29.7.0.tgz",
+            "integrity": "sha512-BrvGY3xZSwEcCzKvKsCi2GgHqDqsYkOP4/by5xCgIwGXQxIEh+8ew3gmrE1y7XRR6LHZIj6yLYnUi/mm2KXKBg==",
+            "dependencies": {
+                "@jest/transform": "^29.7.0",
+                "@types/babel__core": "^7.1.14",
+                "babel-plugin-istanbul": "^6.1.1",
+                "babel-preset-jest": "^29.6.3",
+                "chalk": "^4.0.0",
+                "graceful-fs": "^4.2.9",
+                "slash": "^3.0.0"
+            },
+            "engines": {
+                "node": "^14.15.0 || ^16.10.0 || >=18.0.0"
+            },
+            "peerDependencies": {
+                "@babel/core": "^7.8.0"
+            }
+        },
+        "node_modules/babel-plugin-istanbul": {
```

Comment:
there are a huge amount of changes to package-lock. did mynah-ui add new dependencies? please ensure that your local node_modules is up to date. what command did you run to update the package.json ?

Reply:
I've grabbed the latest master branch, just updated the mynah-ui version to 3.3.2 and called `npm install`. There is no dependency change on the mynah ui for that version update, especially related with babel or related stuff since it doesn't use babel.
Here are the latest 2 changes which fixes this problem:
https://github.com/aws/mynah-ui/commit/af4511bcd5a3e6b4b19a692aded8b4eff68afcae#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519
https://github.com/aws/mynah-ui/commit/261523739c806ee75958219c1cea5b66468aef2c

Reply:
> There is no dependency change on the mynah ui for that version update, especially related with babel or related stuff since it doesn't use babel.

Compare to the old version (3.2.10), the new version (3.3.2) does have changes: https://github.com/aws/mynah-ui/pull/37/files#r1447254162

Why is `babel-jest` in `dependencies` instead of `devDependencies` ? That bloats the production artifact (vsix delivered to the customer).

Reply:
Now it is fixed.